src/cr/cube/cube.py

```diff
@@ -431,6 +449,22 @@ def _is_single_filter_col_cube(self):
         """bool determines if it is a single column filter cube."""
         return self._cube_dict["result"].get("is_single_col_cube", False)
 
+    @lazyproperty
+    def _mean_subreferences(self):
```

Comment:
Do we really need both subreferences and subvariables separately? I think we could ditch at least one of those properties :)